ibis/expr/datatypes.py

```diff
@@ -602,6 +603,38 @@ def __str__(self) -> str:
         )
 
 
+class GeoSpatial(DataType):
+    __slots__ = ()
+
+
+class Point(GeoSpatial):
+    scalar = ir.PointScalar
+    column = ir.PointColumn
+
+    __slots__ = ()
+
+
+class Line(GeoSpatial):
+    scalar = ir.LineScalar
+    column = ir.LineColumn
+
+    __slots__ = ()
+
+
+class Polygon(GeoSpatial):
+    scalar = ir.PolygonScalar
+    column = ir.PolygonColumn
+
+    __slots__ = ()
+
+
+class MultiPolygon(GeoSpatial):
```

Comment:
Can you document these types? I think people may wonder how these types map to PostgreSQL's versions of these, since they are pretty similar. Also, these types should be flexible enough to support Postgres's versions. I think the exercise of going the comparison will be very helpful in determining if this is the right set of types to support databases that support them.

Reply:
it sounds good .. I will also change Line to Linestring. I will also add srid type information.